sql/parser/expr.go

```diff
@@ -359,8 +361,10 @@ func (n *QualifiedName) NormalizeColumnName() error {
 	// Either table.column, table.*, column[array-indirection] or
 	// table.column[array-indirection].
 	switch n.Indirect[0].(type) {
-	case NameIndirection, StarIndirection:
+	case NameIndirection:
 		// Nothing to do.
+	case StarIndirection:
+		n.Indirect[0] = qualifiedStar
```

Comment:
Just a tiny buglet: `*` should be normalized to `"".*`. It only affected error messages which is why I was seeing `kv*` instead of `kv.*`. I suppose I should add a unit test here for normalization of qualified names.